bazel/cc_wrapper.py

```diff
@@ -64,6 +64,18 @@ def main():
         argv.append(arg)
   else:
     argv = sys.argv[1:]
+
+  # Add compiler-specific options
+  if "clang" in compiler:
+    argv.append("-fno-limit-debug-info")
+
+  if "gcc" in compiler:
+    # -Wmaybe-initialized is warning about many uses of absl::optional. Disable
+    # to prevent build breakage. This option does not exist in clang, so setting
+    # it in clang builds causes a build error because of unknown command line
+    # flag.
+    argv.append("-Wno-maybe-uninitialized")
```

Comment:
Same here. IIRC there's also a GCC upstream bug report that would be good to link.

Reply:
Also, per https://github.com/envoyproxy/envoy/issues/2987#issuecomment-378446114 this could be `-Wno-error=maybe-uninitialized` so potential true warnings are still visible but won't break the build.

Reply:
I'm of the opinion that it will just be noise, since we know that it will print in some/many cases that we know are false-positives, so I'd rather disable the warning completely. Anyone else want to weigh in with an opinion?

Reply:
I don't have a strong opinion. I agree that no one will look at the warnings so they are just noise. I'm fine with it as is.